data/building-hours/3-2-krlx-recording-library.yaml

```diff
@@ -4,7 +4,27 @@ category: Radio
 
 schedule:
   - title: Hours
-    hours: []
+    hours:
+      - {days: [Mo], from: '2:00pm', to: '3:00pm'}
+      - {days: [Mo], from: '6:00pm', to: '10:00pm'}
+      - {days: [Tu], from: '1:00pm', to: '3:00pm'}
+      - {days: [Tu], from: '4:00pm', to: '5:00pm'}
+      - {days: [Tu], from: '8:00pm', to: '10:00pm'}
+      - {days: [We], from: '8:00pm', to: '10:00pm'}
+      - {days: [Th], from: '3:00pm', to: '5:00pm'}
+      - {days: [Th], from: '8:00pm', to: '10:00pm'}
+      - {days: [Fr], from: '3:00pm', to: '7:00pm'}
+      - {days: [Sa], from: '11:00am', to: '12:00pm'}
+      - {days: [Sa], from: '1:00pm', to: '5:00pm'}
+      - {days: [Su], from: '7:00pm', to: '10:00pm'}
```

Comment:
This is… I don't think any two days even share a common timeslot. I'm actually kinda impressed.